docs/components/typography/test/index.html

```diff
@@ -0,0 +1,299 @@
+{% extends 'test.njk' %} {% block content %}
+<section>
```

Comment:
Remove section. There is no Shadow DOM that needs testing.

Reply:
removed